org.jacoco.core.test/src/org/jacoco/core/tools/ExecDumpClientTest.java

```diff
@@ -126,6 +126,12 @@ public void testReset() throws IOException {
 		assertTrue(resetRequested);
 	}
 
+	@Test(expected = IOException.class)
+	public void testNopServer() throws IOException {
```

Comment:
@marchof well, IMO no big difference whether to explicitly revert or resolve conflict during merge, but I see that you already did a revert